nucypher/characters/lawful.py

```diff
@@ -312,6 +314,25 @@ def grant():
 
             return Response(json.dumps(response_data), status=200)
 
+        @alice_control.route('/revoke/<policy_encrypting_key>', methods=['DELETE'])
+        def revoke(policy_encrypting_key):
+            policy = drone_alice.active_policies[policy_encrypting_key]
+
+            failed_revocations = drone_alice.revoke(policy)
```

Comment:
What happens if we want to retry later using the `failed_revocations` from `result`? Can we do it?

Reply:
As of right now, we only return the number of failed revocations just to inform the user. They can retry the revoke at a later point.

It just occurred to me though that it's pointless to return the number of failed revocations if the policy was successfully revoked, though.

Reply:
Oh let say this. We revoke a policy, and only half of Ursulas revoke (fail). Then we revoke, and another half revokes. Will it not fail for trying to revoke what was already revoked?
I think, it's totally ok to return `0` for no failed revocations

Reply:
Yeah, that's something I figured out today. Without persistent policies it makes it kinda difficult.

Reply:
At the very least, we can make it so that it doesn't fail if it was already revoked... Or if it was "not found". Does Ursula return different errors for "not found" and "something else"?

Reply:
Yeah, that's the solution I came up with as well.

If you notice, we return the error response. If a policy isn't found, it's a specific error code, so it requires some introspection. Nonetheless, it's still a failed revocation -- especially if an Alice was  not expecting it.

Reply:
Pushed up a commit that makes these changes.